draft-bbs-signatures.md

```diff
@@ -841,6 +845,132 @@ Procedure:
 4. return (a_octets || e_octets || s_octets)
 ```
 
+### OctetsToProof
+
+The `octets_to_proof` operations describes how to parse an octet string representation of a proof, validate it and return the underlying components.
+
+The proof value outputted by this operation consists of the following components, in that order:
+
+1. Three (3) valid points of the G1 subgroup, different from the identity point.
+2. Five (5) integers representing scalars in the range of 1 to r-1 inclusive.
+3. A number of integers representing scalars in the range of 1 to r-1 inclusive, corresponding to the un-revealed from the proof message commitments.
```

Comment:
```suggestion
3. A set of integers representing scalars in the range of 1 to r-1 inclusive, corresponding to the un-revealed from the proof message commitments.
```

Also do we need to clarify the ordering expected here?
```suggestion
3. A number of integers representing scalars in the range of 1 to r-1 inclusive, corresponding to the un-revealed from the proof message commitments.
```

Reply:
> Also do we need to clarify the ordering expected here?

There is no way that i know of for the ordering to be checked here, so there may be no point in adding such description?? The expected "ordering" is kinda "defined" in step 4 in `ProofVerify` i.e., 

```
4. (A', Abar, D, c, e^, r2^, r3^, s^, (m^_j1,...,m^_jU)) = proof
```
Since the functions are just utilities to be used internally and there are no inconsistencies that can arise when used with those core operations, an extra definition of the ordering may not be necessary??